contributor/04-mechanics-of-contributing-script.md

```diff
@@ -26,80 +27,74 @@ Even if you are coming back after a while the teams' ways and members might have
+Add a bit more slack initially. You'll get a feeling about the turn-around times once you work with the host team.
 Often, you will notice a reduction in turn-around time per host team after making a few successful contributions to that host team.
 This is a known effect from the OSS world based on the slow establishment of trust through successful and reliable collaboration.
 
--> Show image of from paper 
 
 I: In your classic teams everyone had an idea of the expectable lead times.
-Within an InnerSource context this might not be the case, either due to large time-zone differences  or you not being available full-time as with your original team even if they are in the same physical location as you are.
+Within an InnerSource context this might not be the case, either due to large time-zone differences or you not being available full-time as with your original team even if they are in the same physical location as you are.
 Thus, to prevent frustration on both sides, impatience and other non-trust-building effects you'll need to explicitly do expectation management with regards to your expected reaction times.
 
 J: This can be as simple as quickly reacting with a "I'll look into it, I won't get to it in the next few days though" to a Trusted Committer's feedback if you know that you'll only be able to come back to them in a few days.
 
-I: As mentioned in the other videos, InnerSource puts huge weight on written communication - in particular when it comes to project decisions.
-Does that imply that in-person communication is forbidden?
-Clearly not: Where written communication shines when it comes to archiving and searchability, in-person communication shines when it comes to communication bandwidth.
+[-> Show illustratory slide on comms bandwidth?]
 
--> Show illustratory slide on comms bandwidth?
 
-J: Try to make time to meet the people behind the names. If possible, try to meet them over your favorite beverage or some food. 
+-> Show trust through rapport slide / overlay
+
+I: Try to make time to meet the people behind the names. If possible, try to meet them over a beverage or some food. 
 If that's not possible having a relaxed informal water-cooler style chat about whatever via your video conference system can provide some of this bandwidth.
 When you're able to hear people speak, when you know their idiosyncrasies remote collaboration will become easier. 
 
-I: The contributions you're going to make will vary in size and scope - sometimes it's just a bug fix, documentation amendment, other times it will be larger feature.
-A time-and-again-proven wisdom from the Open Source arena, especially for large contributions, is to get into contact with the Trusted Committers on their project channel early to get agreement on the user/technical design of the contribution _before_ working on the changes and submitting a pull request.
```

Comment:
I'd leave this in and take out something else instead (passive documentation?).  I feel like getting in touch early is even more important in an enterprise scenario when folks will want to do large features on other repositories and both teams has sprint commitments, etc. that need to be planned in coordination.